debian/rules

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/make -f
 # -*- makefile -*-
-VERSION := $(shell mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -v "\[")
+VERSION := $(shell cat pom.xml |sed '22!d'| cut -d'>' -f2 |cut -d'<' -f1)
```

Comment:
I proposed a similar solution before that was not merged on master, but was
merged on 4.4 . It doesn't use line number.  just check the 4.4 file :-)
On Jul 28, 2015 8:24 AM, "koushik-das" <notifications@github.com> wrote:

> In debian/rules
> <https://github.com/apache/cloudstack/pull/626#discussion_r35620265>:
>
> > @@ -1,6 +1,6 @@
> >  #!/usr/bin/make -f
> >  # -*- makefile -*-
> > -VERSION := $(shell mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -v "\[")
> > +VERSION := $(shell cat pom.xml |sed '22!d'| cut -d'>' -f2 |cut -d'<' -f1)
>
> Is it possible to avoid using the line number (22)? If the line number
> changes for some reason then it won't work.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/cloudstack/pull/626/files#r35620265>.
>